### PR TITLE
(SVACE) SearchBar: remove unnecessary code

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
Two commits:
 * (SVACE) SearchBar: remove unnecessary condition
 *  (SVACE) SearchBar: remove label element

### CODE REVIEW

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Review by TomaszLukawskiSam — CHANGES_REQUESTED** [review]
I need to verify it.

File: src/js/support/mobile/widget/SearchBar.js [hunk 1]
```diff
@@ -662,17 +654,9 @@
 			function inputBlur(self) {
 				var ui = self._ui,
 					inputtedText = ui.input.value,
```
Comment (TomaszLukawskiSam) [inline src/js/support/mobile/widget/SearchBar.js]: 'inputtedText' is assigned a value but never used

**Review by TomaszLukawskiSam — CHANGES_REQUESTED** [review]

File: src/js/support/mobile/widget/SearchBar.js (hunk 1 above)
Comment (grzegorz-czajkowski) [reply src/js/support/mobile/widget/SearchBar.js]: Good catch. Thanks.

**Review by TomaszLukawskiSam — APPROVED** [review]

**Pull request merged**